v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/ExtractionPipeline.scala

```diff
@@ -0,0 +1,245 @@
+package org.broadinstitute.monster.etl.v2f
+
+import caseapp.{AppName, AppVersion, HelpMessage, ProgName}
+import com.spotify.scio.ContextAndArgs
+import com.spotify.scio.{BuildInfo => _, io => _}
+import com.spotify.scio.extra.json._
+import org.broadinstitute.monster.etl.BuildInfo
+
+/**
+  * ETL workflow for converting and tranforming tsv's from v2f.
+  */
+object ExtractionPipeline {
+
+  @AppName("v2f Extraction Pipeline")
+  @AppVersion(BuildInfo.version)
+  @ProgName("org.broadinstitute.monster.etl.v2f.ExtractionPipeline")
+  /**
+    * Command-line arguments for the ETL workflow.
+    *
+    * scio's `ContextAndArgs.typed` delegates down to `caseapp`, which will generate
+    * parsers + help text for these args (as well as Beams' underlying options)
+    */
+  case class Args(
+    @HelpMessage("Directory containing analysis tsv's for v2f")
+    inputDir: String,
+    @HelpMessage("Path of directory where the extracted raw v2f json should be written")
+    outputDir: String
+  )
+
+  /**
+    * pulling in v2f tsv's and converting them to json and performing basic transformations
+    * given the command line arguments
+    */
```

Comment:
Would reword for clarity; something like `Convert V2F TSVs to JSON for ingest into the Data Repository.`